README.md

```diff
@@ -254,9 +254,13 @@ Python:
 instrumentation.opentelemetry.io/inject-python: "true"
 ```
 
-DotNet:
+.NET:
+.NET auto-instrumentation also honors an annotation that will be used to set the .NET runtime version. There are two versions supported `linux-glibc` and `linux-musl`.
+By default `linux-glibc` is used.
```

Comment:
```suggestion
By default `linux-x64` is used.
```

Reply:
I will stop making this rename suggestion here, but, paths and other places should also follow that.

Reply:
176c908030b328b8025ce9afb1b1027bb39c71e0